src/client/components/Editor/index.js

```diff
@@ -150,7 +149,7 @@ class Editor extends React.Component {
       flaggedPhrases,
     } = this.props
 
-    let modifiedText = articleText
+    let modifiedText = articleText.replace(/\n$/g, '\n\n')
```

Comment:
Did you mean to make this a global search (`g`) if you're pinning it to the end with `$`?

Reply:
I think so!  I'll be honest this line was copy pasta'd from the article about the text area hack -- I don't know exactly what it does or why...